test/ts/integration/debt_token.ts

```diff
@@ -1170,4 +1173,47 @@ contract("Debt Token (Integration Tests)", (ACCOUNTS) => {
         //     });
         // });
     });
+
+    describe("#setTokenURI", () => {
+        let tokenId: BigNumber;
+        const tokenURI = "https://www.example.com/image.jpeg";
+
+        before(async () => {
+            tokenId = debtEntries[0].getTokenId();
+        });
+
+        describe("when called by an account that has permission", () => {
+            before(async () => {
+                await debtToken.addAuthorizedTokenURIAgent.sendTransactionAsync(NON_CONTRACT_OWNER);
```

Comment:
Won't this fail if it doesn't come from the multisig?

Reply:
The integration tests for this contract _do not_ use the deployed DebtToken. Example:

```ts
// in integration/debt_token.ts

debtToken = new DebtTokenContract(debtTokenWeb3Contract, TX_DEFAULTS);

await debtRegistry.addAuthorizedInsertAgent.sendTransactionAsync(debtToken.address, {
            from: CONTRACT_OWNER,
});
await debtRegistry.addAuthorizedEditAgent.sendTransactionAsync(debtToken.address, {
            from: CONTRACT_OWNER,
});
```

If this is an issue we should change it in a separate PR. It was weird when I was it.

Reply:
Interesting -- we should _definitely_ fix that.  Can you make a task and add it to our Pivotal queue before closed beta?